pkgs/by-name/dt/dtv-scan-tables/package.nix

```
@@ -4,13 +4,13 @@
 , v4l-utils
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "dtv-scan-tables";
-  version = "2022-04-30-57ed29822750";
+  version = "2024-03-24-7098bdd27548";
 
   src = fetchurl {
-    url = "https://linuxtv.org/downloads/${pname}/${pname}-${version}.tar.bz2";
-    hash = "sha256-amJoqjkkWTePo6E5IvwBWj+mP/gi9LDWTTPXE1Cm7J4=";
+    url = "https://linuxtv.org/downloads/${finalAttrs.pname}/${finalAttrs.pname}-${finalAttrs.version}.tar.bz2";
```

```suggestion
    url = "https://linuxtv.org/downloads/dtv-scan-tables/dtv-scan-tables-${finalAttrs.version}.tar.bz2";
```
The url will always be wrong unless the string matches this exactly, which makes substituting it kinda pointless

---

Done, thanks.  And rebased onto current `nixos-unstable`.